urls/week2-raye

```diff
@@ -34,7 +34,6 @@ https://www.wthr.com/article/live-nation-postpone-all-tours-due-coronavirus
 https://www.aljazeera.com/news/2020/03/coronavirus-medical-staff-call-public-stay-home-200320091536804.html
```

Comment:
It's okay if you modify this file for your own local work, but you shouldn't commit your modifications to the repo.